src/com/atlas/controller/TitleController.java

```diff
@@ -18,18 +23,54 @@
 public class TitleController {
 
 	@Autowired
+	@Qualifier("titleServiceImpl")
 	private TitleService titleService;
 
 	@RequestMapping(method = RequestMethod.GET, produces=MediaType.APPLICATION_JSON_VALUE)
 	public List<Title> getAllTitles() {
 		return this.titleService.listTitles();
 	}
+	
+	@RequestMapping(value="{id}",method = RequestMethod.GET, produces=MediaType.APPLICATION_JSON_VALUE)
+	public Title getTitleById(@PathVariable("id") int id) throws MovieNotFound{
+		return this.titleService.getTitleById(id);
+	}
 
 	@RequestMapping(method = RequestMethod.POST, produces=MediaType.APPLICATION_JSON_VALUE,
 			consumes=MediaType.APPLICATION_JSON_VALUE)
-	public Title addTitle(@RequestBody Title t) {
-		this.titleService.addTitle(t);
-		return t;
+	public Title addTitle(@RequestBody Title t) throws MovieBadRequest{
```

Comment:
I dont think there is a need to return the object, 200 Status code is good enough